doc-content/kogito-docs/src/main/asciidoc/openshift/con_kogito-on-ocp.adoc

```diff
@@ -0,0 +1,10 @@
+[id='con_kogito-on-ocp']
+
+= {PRODUCT} on {OPENSHIFT}
+
+You can deploy {PRODUCT} services on {OPENSHIFT} for cloud implementation. In this solution, {PRODUCT} services are deployed as OpenShift pods that you can scale up and down individually to provide as few or as many containers as required for a particular service. You can use standard OpenShift methods to manage the pods and balance the load.
+
+To help you deploy your services on OpenShift, {PRODUCT} provides the following resources:
+
+* *{PRODUCT} Operator*: An operator that guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can pull the project, build it (including native compilation, if applicable), and deploy the resulting services.
```

Comment:
same than before about the operator pulling the code.

Reply:
Using comment above.